cognite/client/utils/_pandas_helpers.py

```diff
@@ -37,6 +57,13 @@ def notebook_display_with_fallback(inst: Union[T_CogniteResource, T_CogniteResou
         return str(inst)
 
 
+def convert_nullable_int_cols(df: pd.DataFrame, camel_case: bool) -> pd.DataFrame:
```

Comment:
```suggestion
def convert_nullable_int_cols_to_int64(df: pd.DataFrame, camel_case: bool) -> pd.DataFrame:
```

Reply:
This one is actually wrong; "int64" is "_not nullable int_", while "Int64" is nullable, so I'm keeping the original 🤙